vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
```

Comment:
**nitpick**: The comment on this line seems unnecessary. Consider removing it to keep the code clean.